app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
```

Comment:
- [ ] Dear @haftamudesta, Just as mentioned by the previous reviewer, Please consider leaving the `index` and `show` actions/methods empty at this point. Currently, we are not passing a user ID to the show route and this keeps throwing an error.  I strongly suggest sticking to the requirements at hand, and at this point it's not required to have a user in our param.

### the error
![Screen Shot 2023-08-28 at 11 29 43 PM](https://github.com/haftamudesta/Blog-App/assets/69077061/1de24437-3fbd-4ce3-8fc3-c1a2211f3dd3)